src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -65,30 +51,36 @@ void M(int [|p|])
         [Fact]
         public async Task Parameter_Unused()
         {
-            await TestDiagnosticsAsync(
-                """
+            await VerifyAnalyzerAsync("""
                 class C
                 {
-                    void M(int [|p|])
+                    void M(int {|IDE0060:p|})
```

Comment:
📝 I mentioned in another commit that literal diagnostic IDs should reference the named constants. Note that this would _not_ apply for IDs appearing with the markup syntax `{|IDE####:...|}`.